src/urbit/mar/chess/update.hoon

```diff
@@ -40,6 +40,11 @@
         :~  ['chessUpdate' [%s 'draw-declined']]
             ['gameID' [%s (scot %da game-id.upd)]]
         ==
+      %draw-claim-available
+        %-  pairs:enjs
+        :~  ['chessUpdate' [%s 'draw-claim-available']]
+            ['gameID' [%s (scot %da game-id.upd)]]
+        ==
```

Comment:
As with the `special-draw` poke in `/app`, I think this should be something like `threefold-draw-claim-available` if I'm not missing something.

Reply:
See above - this was my call for merging threefold repetition and the 50-move rule into one logic flow